code/index.html

```diff
@@ -6,16 +6,31 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Project GitHub Tracker</title>
   <link rel="stylesheet" href="./style.css" />
+  <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+  
 </head>
 <body>
-  <h1>GitHub Tracker</h1>
-  <h2>Projects:</h2>
-  <main id="projects"></main>
+  <header class="hero">  
+    <div id="userDataWrapper" class="user-data-wrapper"></div>
+    <section class="img-wrapper" id="imgWrapper">  </section>
+  </header>
 
+  <main class="project-wrapper" id="projects"></main>
+   
   <!-- This will be used to draw the chart 👇 -->
-  <canvas id="chart"></canvas>
+  <section class="chart-box">
+    <divc class="chart-wrapper" style="position: relative;">
+      <canvas  id="chart"></canvas>
+  </div>
+</section>
+
+<footer>
+  <p> images from unsplash.com</p>
+</footer>
 
-  <script src="./script.js"></script>
   <script src="./chart.js"></script>
+  <script src="./secret.js"></script>
```

Comment:
I would suggest removing this script tag for the secret js file. I don't believe you have a secret js file?